lib/libc/stdio/gets_s.c

```diff
@@ -47,31 +47,47 @@
 static inline char *
 _gets_s(char *buf, rsize_t n)
 {
-	int c;
+	int c, signal;
 	char *s;
+	/* 
+ 	 * This modifications prevents error qhen the user type exactly N - 1
```

Comment:
Should that `when` be `when`?

Reply:
Yes, typo wrong

Reply:
This is the final version, but I edited it in the GitHub editor and some things are wrong, like tabs and spaces.